modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/terraformboot/TerraformBootScriptValidator.java

```diff
@@ -128,4 +123,15 @@ private List<String> getFilesByOcl(Deployment deployment) {
         String deployer = deployment.getDeployer();
         return Collections.singletonList(deployer);
     }
+
+    private UUID getRequestId() {
```

Comment:
i think when we send requests to terraform-boot or tofu-maker the order ID is present. It is better to use that here.

Reply:
This method is used in register service template, the order id always is null.